public/app/features/explore/Logs/Logs.tsx

```diff
@@ -553,6 +553,9 @@ class UnthemedLogs extends PureComponent<Props, State> {
     const navigationRange = this.createNavigationRange(logRows);
 
     const scanText = scanRange ? `Scanning ${rangeUtil.describeTimeRange(scanRange)}` : 'Scanning...';
+    const numberOfLogVolumes = logsVolumeData?.data?.length ?? 1;
```

Comment:
Checking the number of results is bad as it's constantly changing as queries are executed, but if we can determine the number of queries that have log volumes, we might be able to better reserve the space.

Also it looks like each datasource only gets one, so it's not the number of unique queries, but the number of logs applicable datasources used.